.github/workflows/ci-publish.yml

```diff
@@ -0,0 +1,37 @@
+---
+
+name: ci-publish
```

Comment:
Can this workflow name changed? Maybe `publish-docker`. I personally see `ci` for building and testing but in this particular case it generates and publish the docker images